### PR TITLE
fix: move to the details page after completing collector creation

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue [hunk 1]
```diff
@@ -150,17 +150,24 @@ const handleConfirmCreateCollector = async () => {
             collector_id: state.createdCollectorId,
         });
         showSuccessMessage(i18n.t('INVENTORY.COLLECTOR.CREATE.ALT_S_COLLECT_EXECUTION'), '');
-        handleRouteToCollectorList();
+        handleRouteToCollectorDetailPage();
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue:153]: suggestion: We decided not to use event handler in other function!
But we decided to allow the case of using normal function as a event handler.
So, what about to change the name of function `handleRouteToCollectorDetailPage` to `goToCollectorDetailPage`?

**Review by WANZARGEN — APPROVED** [review]
Good!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue:153]: WT... 
👍

**Pull request merged**